src/file_worker.py

```diff
@@ -86,7 +93,8 @@ def __init__(self, appConfig=None, requestHeaders=None):
             self.uuid_api_url = appConfig['UUID_API_URL']
             self.entity_api_url = appConfig['ENTITY_API_URL']
```

Comment:
@kburke we didn't capture this in the initial release, can you add `strip('/')` to these configurable URLs to remove the trailing slash if by any chance they have one. I noticed in the actual API calls you specified the path separator `/` after the base URL.